project/settings/local-dist.py

```diff
@@ -18,3 +18,5 @@
 EMAIL_BACKEND = (
     f"django.core.mail.backends.{'console' if DEBUG else 'smtp'}.EmailBackend"
 )
+
+# SESSION_COOKIE_SECURE= False # For local debugging
```

Comment:
This shouldn't *in general* be necessary - I'd hold off on incorporating it as part of internationalization, but we should look at it when we update setup instructions

Reply:
My ubuntu workstation won't offer https:/// connections on localhost without me configuring a certificate. It took me quite a long time that it was the lack of a stateful session that was causing all manner of odd problems with the authentication... it seemed to authenticate, but then acted like it didn't know me on the new page. I think I'd changed a few hundred lines by the time I discovered it was just this one. So, I agree - not at all related to internationalization, and would be well placed in the instructions.  If you'd like Ubuntu 18.04 setup instructions I've been through a few iterations and now have a pretty tidy list I think.

Reply:
@rhodricusack is there a reason why you can't configure a local certificate? `mkcert` might be able to do most of the legwork for you, and then you'd just have to configure `runserver_plus` with the `--cert-file` flag.